master/buildbot/scripts/start.py

```diff
@@ -129,10 +129,37 @@ def launch(config):
             pidfile.write("{0}".format(proc.pid))
 
 
+def py2Warning(config):
+    if sys.version[0] == '2' and not os.path.exists(os.path.join(config['basedir'], "no_python2_warning")):
```

Comment:
Maybe we should add a way to run buildbot tests by checking whether some key like `BUILDBOT_RUNNING_PYTHON_27_TESTS` is in os.environ? That way we could still run tests on python 2.7 right until we actually drop support.